packages/dx-react-grid-bootstrap3/package.json

```diff
@@ -56,6 +56,7 @@
     "output": "../../shippable/testresults/dx-react-grid-bootstrap3.xml"
   },
   "devDependencies": {
+    "@devexpress/dx-react-grid": "1.0.0-alpha.2",
```

Comment:
Looks like this change was introduced while merging the master branch. I'll fix it, thanks.